src/main/groovy/org/mockito/release/version/DefaultVersionInfo.java

```diff
@@ -103,11 +107,12 @@ public DefaultVersionInfo bumpVersion(boolean updateNotable) {
     }
 
     public Collection<String> getNotableVersions() {
+        //TODO if by the end of Q3 (9/30/17) we don't find the concept of 'notable versions' useful
+        //we should delete 'notableVersions' code from this class
         return notableVersions;
```

Comment:
Maybe is it better to delete it now and add it later (if needed), because of compatibility after 1.0?